Update the perl-SDL check to accept 2.x versions.

Make sure we update the DATADIR stuff before building in 
subdirs otherwise the lib file gets copied to blib first 
and the final binary isn't fixed up properly.

--- Makefile
+++ Makefile
@@ -11,6 +11,3 @@
 	@if ! perl -e 'use SDL'; then echo -e "\n    *** I need perl-SDL installed"; false; fi
-	@if ! perl -e 'use SDL; ($$mj, $$mn, $$mc) = split /\./, $$SDL::VERSION; exit 1 if $$mj<1 || $$mn<19'; then echo -e "\n    *** I need perl-SDL version 1.19.0 or upper"; false; fi
+	@if ! perl -e 'use SDL; ($$mj, $$mn, $$mc) = split /\./, $$SDL::VERSION; exit 1 if $$mj<1 || ($$mj==1 && $$mn<19)'; then echo -e "\n    *** I need perl-SDL version 1.19.0 or upper"; false; fi
-	@for n in . $(DIRS); do \
-		[ "$$n" = "." ] || $(MAKE) -C $$n ;\
-	done
 	@if [ ! -d save_virgin ]; then mkdir save_virgin; cp c_stuff/lib/fb_stuff.pm save_virgin; fi
@@ -18,2 +15,5 @@
 	perl -pi -e 's|\@DATADIR\@|$(DATADIR)|' c_stuff/lib/fb_stuff.pm
+	@for n in . $(DIRS); do \
+		[ "$$n" = "." ] || $(MAKE) -C $$n ;\
+	done
 
